CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unpublished
 * larva-tokens - Add Robb Report tokens
+* writeJSON will fail on syntax errors
```

Comment:
Prefix entries in the Changelog with `{package} - description of change`. In this case, it would be the `larva` package. And maybe add this to the descriptive text at the top of the file?